addons/inventory_report_generator/models/dynamic_inventory_report.py

```
@@ -374,9 +373,199 @@ def _accumulate(mls, is_before: bool):
+            return {
+                "product": {},
+                "lines": [],
+                "summary": {},
+            }
+
+        ctx = self._get_base_domains(rec)
+        loc_ids = ctx["loc_ids"]
+        qty_field = self._ml_qty_field()
+        SML = self.env["stock.move.line"].sudo()
+
+        domain = list(ctx["period_domain"]) + [("product_id", "=", int(product_id))]
```

Ruff (0.15.11)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)